src/components/Card/MarketplaceCard.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div
+    :class="{
+      'unnnic-card-marketplace': true,
+      'unnnic-card-marketplace__clickable': clickable,
+      'unnnic-card-marketplace__add': typeAction === 'add',
+      'unnnic-card-marketplace__config': typeAction === 'config',
+      'unnnic-card-marketplace__edit': typeAction === 'edit',
+    }"
+    @click="cardClick"
+  >
+    <div :class="['unnnic-card-marketplace__icons']">
+      <img class="unnnic-card-marketplace__icons__app" :src="iconSrc" />
+      <unnnic-button
+        class="unnnic-card-marketplace__icons__button"
+        size="small"
+        type="secondary"
+        @click.stop="buttonClick"
+        :iconCenter="getIcon()"
+      />
+    </div>
+    <div class="unnnic-card-marketplace__content">
+      <div class="unnnic-card-marketplace__content__title">{{ title }}</div>
+      <div class="unnnic-card-marketplace__content__description" :title="description">
+        {{ description }}
+      </div>
+      <div v-if="typeAction === 'add'" class="unnnic-card-marketplace__content__rating">
+        <unnnic-icon-svg
+          class="unnnic-card-marketplace__content__rating__star"
+          scheme="feedback-yellow"
+          icon="rating-star-1"
+          size="ant"
+        />
+        <div class="unnnic-card-marketplace__content__rating__value">{{ rating }}</div>
+        <div class="unnnic-card-marketplace__content__rating__comments">({{ comments }})</div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import UnnnicButton from '../Button/Button.vue';
+import UnnnicIconSvg from '../Icon-svg.vue';
+
+export default {
+  name: 'unnnic-card',
+  components: { UnnnicButton, UnnnicIconSvg },
+  props: {
+    title: {
+      type: String,
+      default: null,
+    },
+    description: {
+      type: String,
+      default: null,
+    },
+    comments: {
+      type: String,
+      default: null,
+    },
+    clickable: {
+      type: Boolean,
+      default: false,
+    },
+    iconSrc: {
+      type: String,
+      default: null,
+    },
+    rating: {
+      type: Number,
+      default: 0,
+    },
+    typeAction: {
+      type: String,
+      default: null,
+    },
+    buttonAction: {
+      type: Function,
+      default: () => {},
+    },
+  },
+
+  methods: {
+    buttonClick() {
+      if (this.typeAction === 'add') {
+        this.buttonAction();
+      } else {
+        this.$emit('openModal');
+      }
+    },
+    cardClick() {
+      this.$emit('openModal');
+    },
+    getIcon() {
+      switch (this.typeAction) {
+        case 'add':
+          return 'add-1';
+        case 'config':
+          return 'cog-1';
+        case 'edit':
+          return 'pencil-write-1';
+        default:
+          return 'add-1';
+      }
+    },
```

Comment:
And if `getIcon` was just a computed called `icon` due it is like a "read-only variable"?